tests/scripts/visual-coverage.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+start-storybook -p 6006 --ci &
+JOB_ID=$(echo $!) #get the background job ID
+
+# wait for a locally served Storybook
+attempt_counter=0
+max_attempts=30
+until $(curl --output /dev/null --silent --head http://localhost:6006)
+do
+  if [ ${attempt_counter} -eq ${max_attempts} ];then
+    echo "Max attempts reached"
+    exit 1
```

Comment:
have you tested if this would fail as expected? I'm wondering if this would just exit the foreground command and continue running the background storybook command

Reply:
I tested it by waiting for another port number and it exited the program. The assumption is the background storybook command either fails or if it continues then it wouldn't exit in here anyway. But for a proper clean up, I will update it to kill the background job as well!